charter.md

```diff
@@ -0,0 +1,336 @@
+    - [Business Function Owner](#business-function-owner)
+  - [Technology Stack (brainstorm)](#technology-stack-brainstorm)
+    - [Architecture](#architecture)
+    - [GitOps for $allTheThings](#gitops-for-allthethings)
+    - [Platform Composition & Abstraction](#platform-composition--abstraction)
+    - [Ingress: NGINX](#ingress-nginx)
+    - [Prometheus operator](#prometheus-operator)
+    - [Service Mesh: Linkerd](#service-mesh-linkerd)
+  - [workloads: Objects of Observation (wOoO)](#workloads-objects-of-observation-wooo)
+    - [CNCF Projects: Graduated (16)](#cncf-projects-graduated-16)
+    - [Incubating (25)](#incubating-25)
+    - [CNCF Projects: Sandbox (66)](#cncf-projects-sandbox-66)
+    - [Potential example workloads](#potential-example-workloads)
+    - ["Sock Shop"](#sock-shop)
+    - [.net core example (from Microsoft)](#net-core-example-from-microsoft)
+    - [Linkerd Emojivote](#linkerd-emojivote)
+    - [podtato-head](#podtato-head)
+    - [APM Game](#apm-game)
+  - [Appendices & Resources](#appendices--resources)
+    - [Mandate (from TAG Charter)](#mandate-from-tag-charter)
+      - [Mission](#mission)
+      - [Areas considered in Scope](#areas-considered-in-scope)
+  - [Meeting Notes](#meeting-notes)
+    - [2021-12-16](#2021-12-16)
+      - [Actions](#actions)
+    - [2021-12-09](#2021-12-09)
+      - [Actions](#actions-1)
+
+## Mission Statement
+
+For cloud native workloads, observability is a key requirement, yet it’s a broad term and there are many tools and approaches available to achieve it.
+
+While the [whitepaper](https://github.com/cncf/tag-observability/blob/main/whitepaper.md) by the [CNCF TAG Observability](https://github.com/cncf/tag-observability) covers the theoretical foundations of modern observability, practitioners need more guidance on how to achieve it in real life scenarios.
+
+This working group aims to provide:
+
+- a set suite of realistic sample applications, which employ observability tools to demonstrate how practitioners can gain insight and understanding.
+- actionable, best-practice guidance on how to apply observability to real-life workloads. Observed workloads will be CNCF Projects.
+
+_A distinct section requiring intentional navigation by the User shall be provided for open source projects which are not CNCF Projects if this opportunity arises._
+
+## Interested Parties
+
+- Matt Young ([halcyondude@gmail.com](mailto:halcyondude@gmail.com))
+- Ivan Santos ([pragmaticivan@gmail.com](mailto:pragmaticivan@gmail.com), EverQuote)
+- Ken Finnigan ([ken@kenfinnigan.me](mailto:ken@kenfinnigan.me), Workday)
+- Michael Hausenblas ([mh9@o11y.engineering](mailto:mh9@o11y.engineering), AWS)
+- Ryan Perry ([rperry2174@gmail.com](mailto:rperry2174@gmail.com), Pyroscope)
+- Daniel Khan ([daniel@khan.io](mailto:daniel@khan.io), Elastic)
+- Severin Neumann ([severin.neumann@altmuehlnet.de](mailto:severin.neumann@altmuehlnet.de), Cisco)
+
+## Discussion(s) thus far:
```

Comment:
Either expand on this section or remove. As it stands, I doubt the bullet list is helpful for the reader.

Reply:
agreed and removed.  Left a few concrete ddetails from gitops wg and cncf.

I think this document should be cleaved into a charter that is only mission statement + charter, and the rest can seed a design TLD.